src/erdstall.ts

```diff
@@ -62,33 +63,34 @@ export interface Leaver extends Exiter, Withdrawer {
 	leave(): Promise<Stages<Promise<ethers.ContractTransaction>>>;
 }
 
+export interface BoundSubscriber {
+	subscribeSelf(): Promise<void>;
+}
+
 export interface Subscriber {
-	subscribe(): Promise<void>;
+	subscribe(who?: Address):Promise<void>;
+}
+
+export interface Onboarder {
 	onboard(): Promise<void>;
 }
 
-export interface Erdstall
+export interface ErdstallClient
 	extends Watcher,
+		Subscriber {
+	initialize(): Promise<void>;
```

Comment:
Please add comment when `initialize` should be called. Is it before or after `subscribe`?